test/TestProjects/TypeSchemaMapping/TypeSchemaMapping.json

```diff
@@ -191,6 +192,25 @@
       },
       "x-namespace": "Very.Custom.Namespace.From.Swagger",
       "x-accessibility": "internal"
+    },
+    "ModelWithArrayOfEnum": {
+      "type": "object",
+      "properties": {
+        "ArrayOfEnum": {
+          "type": "array",
+          "items": {
+            "type": "string",
```

Comment:
Nit: while you don't explicitly support it now, it might be good to add `"x-nullable": false` because services *should* be doing this, which was the case for Search (they did)...unless they really do want to allow for null values.

Reply:
`"x-nullable": false` is considered the default. We'll add coverage for that when adding support.

Reply:
Granted, so this should break if not modified later (in swagger) but the support is added; hence, why I was suggesting it now. But either works. 🙂